pkg/webservice/handlers.go

```diff
@@ -275,6 +276,7 @@ func getPartitionJSON(partition *scheduler.PartitionContext) *dao.PartitionDAOIn
 
 	queueDAOInfo := partition.GetQueueInfos()
 
+	// this method is used by a deprecated API, so it keeps returning normalized name
```

Comment:
I am under the assumption that we would be doing follow up changes in web app/UI component as well similar to YUNIKORN-794. If it it so, then, should we take care of this place as well?

Reply:
You are right. will copy that